share/doc/wake/quickref.md

```diff
@@ -249,17 +249,20 @@ Runners are created using `makeRunner`. `makeRunner` is defined as follows:
 
 Environment packages can be added to a workspace to provide tools (using runners) for running the workflow. They are composed of Wake files that define and publish runners. Environment packages should be defined for each unique running environment to decouple Wake rules from how tools are installed in each environment. You can check out [environment-example-sifive](https://github.com/sifive/environment-example-sifive) as an example of an environment package that contains runners for Wake-based workflows. 
 
-#### sources and Path objects
+#### Sources and `Path` objects
 
-Sources are the set of files in git.
+Sources are the set of files in a Git repository. A `Path` in Wake is either a) a file in Git or b) a file produced from a build step. The `Path` type has no public constructor to help enforce this. To get a `Path` value, use the
 
-A Path is either a) a file in git or b) a file produced from a build step
+* `source: (file: String) => Result Path Error`
+* `sources: (dir: String) => (filterRegexp: RegExp) => Result (List Path) Error`
 
-To get a path, use the source function. For example, all the header files in this directory:
+functions. For example, to get all the header files in the current repository:
 
+```wake
+sources here `.*\.h`
 ```
-source `.*\.h` here
-```
+
+Note that the directory given to `sources` is always relative to the root directory of the current Git repository and not the current directory. `here: String = "."` is just a helper value.
```

Comment:
Thank you for the corrections! (As mentioned over Zoom but just mentioning here for completeness) I thought it was a value given the output of `wake -vx 'here'`, but I should have checked it inside some nested files as having an alias for just `"."` wouldn't be that useful. Haha.

I went ahead and put in your examples into the reference as well, in addition to updating the explanation [here](https://github.com/sifive/wake/pull/1009/commits/4128f7caa46084157a37e748456da2c13b9612f1).

Is the line

> Note that the directory given to `sources` is always relative to the root directory of the current Git repository and not the current directory.

accurate? I wasn't for sure since it reaches out to `prim "sources"`.

Reply:
Hmm, I still think it is more appropriate to say "relative to the wake root". I don't think that we enforce the wake root to be the same as the git route. Though I think they are the same by convention.

Reply:
Thanks for the clarification! Sam and I happened to hit upon this today as well, where he pointed out the `.wakeroot` file in Federation. I'll update this shortly and maybe make another PR later on to elaborate on the Wake root and file finding situation once I learn it more thoroughly, just to capture it somewhere.